proto/cmp/types/v1alpha1/product_list.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package cmp.types.v1alpha1;
+
+// Product list code and type
+//
+// This is being used in requests to specify the a list of products to be included in the reponse
+//
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha1/product-list.proto.dot.xs.svg)
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha1/product-list.proto.dot.svg)
```

Comment:
The file name here not correct, it should use the exact name of the `.proto` file. In this case it should be `product_list.proto`. **But** because I suggested to change the message type name to `ProductCode`, I will also suggest to change the file name to `product_code.proto`

```suggestion
// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha1/product_list.proto.dot.xs.svg)
// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha1/product_list.proto.dot.svg)
```

Reply:
Agreed and implemented